docker/transformers-all-latest-gpu/Dockerfile

```diff
@@ -3,6 +3,9 @@ LABEL maintainer="Hugging Face"
 
 ARG DEBIAN_FRONTEND=noninteractive
 
+# Used to read variables from `~/.profile` (to pass dynamic created variables between RUN commands)
+SHELL ["sh", "-lc"]
```

Comment:
Dockfile is really limited, and I really need to pass some variables between `RUN` command.